core/src/test/java/com/google/instrumentation/trace/SamplersTest.java

```diff
@@ -24,29 +24,33 @@
 /** Unit tests for {@link Samplers}. */
 @RunWith(JUnit4.class)
 public class SamplersTest {
+  private static final Random random = new Random(1234);
+  private static final TraceId traceId = TraceId.generateRandomId(random);
+  private static final SpanId parentSpanId = SpanId.generateRandomId(random);
+  private static final SpanId spanId = SpanId.generateRandomId(random);
```

Comment:
I think it would be better if these fields were non-static, for the same reason as above.